.eslintrc.js

```diff
@@ -16,7 +16,8 @@ module.exports = {
     browser: true
   },
   rules: {
-    'ember/no-jquery': 'error'
+    'ember/no-jquery': 'error',
+    'ember/no-observers': 'warn'
```

Comment:
We need this so that builds don't fail until we replace observer usage
I tried and failed here https://github.com/kategengler/ember-feature-flags/pull/98

Reply:
I don't understand why you needed to change the lint rules when you don't introduce any observers here and have not updated dependencies?

Reply:
I don't quite understand either, I would have thought this lint issue would have showed up in whatever PR added the lint rule

But you can see from the failed builds in the earlier commits of this PR that it's due to this lint rule.
https://travis-ci.org/github/kategengler/ember-feature-flags/jobs/738722244

I can remove the lint warn override if you like

```
not ok 4 Chrome 62.0 - [0 ms] - ESLint | addon: addon/helpers/feature-flag.js
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/dummy.js:58:12)
                at runTest (http://localhost:7357/assets/test-support.js:5669:32)
                at Test.run (http://localhost:7357/assets/test-support.js:5657:8)
                at http://localhost:7357/assets/test-support.js:5874:15
                at processTaskQueue (http://localhost:7357/assets/test-support.js:5264:26)
                at http://localhost:7357/assets/test-support.js:5268:12
        message: >
            addon/helpers/feature-flag.js should pass ESLint
            
            14:5 - Don't use observers (ember/no-observers)
```

Reply:
Additionally you can see that the build failed on the 6.0 commit - opening the travis log shows it was this lint rule as well.

![Screenshot from 2020-10-26 10-13-02](https://user-images.githubusercontent.com/1049837/97122741-e2e0e700-1773-11eb-80dd-a7d22dcde6be.png)

Reply:
It looks like it started failing on the cron in the scenarios that float dependencies. Instead of adding the rule here, would you mind adding it where the observers are used, using eslint-disable? Something like `/* eslint-disable ember/no-observers */` and `/* eslint-enable ember/no-observers */ wrapped around the usage of observers?